exercises/G-numbers/exercise.js

```diff
@@ -1 +1,8 @@
-// Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+let numberOfStudents = 78;
+let numberOfMentors = 7;
+let sum = numberOfStudents + numberOfMentors;
+
+console.log("Number of students: "+numberOfStudents);
```

Comment:
please make sure the spacing is consistent so that the code is easier to read, i.e. there must be a space beween closing double quotes and the plus sign as well as between the plus sign and the variable ``numberOfStudents``